resources/find_nonindexed/get_nonindexed_files.py

```diff
@@ -1,22 +1,25 @@
+#!/usr/bin/env python3
+# get_nonindexed_files.py
 """Utility to get files that have not been indexed from a traverse file."""
 
-
 import argparse
 import concurrent.futures
 import json
 import logging
 from typing import List
 
 import coloredlogs  # type: ignore[import]
-import more_itertools as mit  # type: ignore[import]
-from rest_tools.client import RestClient  # type: ignore[import]
+import more_itertools as mit
+from rest_tools.client import ClientCredentialsAuth
 
 
-def _check_fpaths(fpaths: List[str], token: str, thread_id: int) -> List[str]:
+def _check_fpaths(fpaths: List[str], client_secret: str, thread_id: int) -> List[str]:
     # setup
-    rc = RestClient(
-        "https://file-catalog.icecube.wisc.edu/",
-        token=token,
+    rc = ClientCredentialsAuth(
+        address="https://file-catalog.icecube.wisc.edu/",
+        token_url="https://keycloak.icecube.wisc.edu/auth/realms/IceCube",
+        client_id="file-catalog",
+        client_secret=client_secret,
         timeout=60 * 60,  # 1 hour
         retries=24,  # 1 day
```

Comment:
The timeout and retries seem exceedingly large?  I know of no cases where you'd want to sit there for a day instead of failing (and trying to run this again later).

Reply:
I wrote this part and cannot remember the reasoning. It could've been a misunderstanding of how those two arguments work--hard to say. It'd probably be best to require these as CL args